src/main/java/com/codegym/shopyy/model/Address.java

```diff
@@ -0,0 +1,39 @@
+package com.codegym.shopyy.model;
+
+import jakarta.persistence.*;
```

Comment:
use import-specific library instead of import .*

Reply:
complete